schema/mysql/schema.sql

```diff
@@ -612,6 +613,7 @@ CREATE TABLE downtime (
   is_in_effect enum('n', 'y') NOT NULL,
   start_time bigint unsigned DEFAULT NULL COMMENT 'Time when the host went into a problem state during the downtimes timeframe',
   end_time bigint unsigned DEFAULT NULL COMMENT 'Problem state assumed: scheduled_end_time if fixed, start_time + flexible_duration otherwise',
+  duration bigint unsigned NOT NULL COMMENT 'Duration of the downtime: When the downtime is flexible, this is the same as flexible_duration otherwise scheduled_end_time - scheduled_start_time',
```

Comment:
According to https://github.com/Icinga/icinga2/issues/9049, this column will be used for sorting. Does just adding that as a column without an index on it already improve things there?

Reply:
Yeah, it does. But an index for this is added with this one. https://github.com/Icinga/icingadb/pull/397

Reply:
Okay, so then that PR is also blocked by this one as it adds indices on columns only introduced here?